lib/osutil/symlink_windows.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package osutil
+
+import (
+	"os"
+	"path/filepath"
+	"syscall"
+)
+
+// DebugSymlinkForTestsOnly is os.Symlink taken from the 1.9.2 stdlib,
+// hacked with the SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE flag to
+// create symlinks when not elevated.
+//
+// This is not and should not be used in Syncthing code, hence the
+// cumbersome name to make it obvious if this ever leaks. Nonetheless it's
+// useful in tests.
+func DebugSymlinkForTestsOnly(oldname, newname string) error {
+	// CreateSymbolicLink is not supported before Windows Vista
+	if syscall.LoadCreateSymbolicLink() != nil {
+		return &os.LinkError{"symlink", oldname, newname, syscall.EWINDOWS}
+	}
+
+	// '/' does not work in link's content
+	oldname = filepath.FromSlash(oldname)
+
+	// need the exact location of the oldname when its relative to determine if its a directory
```

Comment:
If you're really worried about the dictionary attacks: Its->it's

Reply:
:D none of this code is written by us.

Reply:
Oh damn, I don't read the copyright stuff. But that's a relieve, because that functions is s**t xD